src/pages/documentation/current-sdk-versions.md

```diff
@@ -118,6 +118,11 @@ If you are currently using the ACP-prefixed React Native libraries, please see [
 | [Places](./places/index.md) | [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepplaces.svg?color=green&label=%40adobe%2Freact-native-aepplaces&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepplaces) |
 | [Target](../adobe-target/index.md) | [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aeptarget.svg?color=green&label=%40adobe%2Freact-native-aeptarget&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aeptarget) |
 | [Campaign Classic](./adobe-campaign-classic/index.md) | [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepcampaignclassic.svg?color=green&label=%40adobe%2Freact-native-aepcampaignclassic&logo=npm&style=flat-square)](https://badge.fury.io/js/%40adobe%2Freact-native-aepcampaignclassic) |
+| Adobe Analytics| Not Supported - Analytics workflows supported through [Edge](../adobe-analytics/migrate-to-edge-network.md) or [Edge Bridge](../adobe-analytics/migrate-to-edge-network.md) extensions. |
```

Comment:
I am pointing both Edge and Edge Bridge extensions to the migration doc, which is the location Analytics user should go through first to learn about Edge and Edge Bridge.
Or I should point them to Edge and Edge Bridge GitHub directly?

Reply:
one other option is to create only one link, reading:
Not supported, Analytics workflows supported through Edge or Edge Bridge extensions, see [guide to migrate to Edge Network](../adobe-analytics/migrate-to-edge-network.md).

and same for flutter